### DISCUSSION TITLE
XY Plot

### DISCUSSION THREAD

**Discussion by klaaslusterbroodvan** [patrykandpatrick/vico#458] [Questions]
I wonder if it is possible to make an XY plot like you can do with MPAndroidChart (which otherwise is not Compose friendly). I tried to look at libraries for Android Compose that support basic 2D X-Y plots, but somehow Vico does not seem to support this, or am I missing something?

**Comment by patrickmichalik** [comment]
Hello! You can create a scatter chart by using a line chart with points and zero-width lines.
